Sources/SwifterSwift/Foundation/NSAttributedStringExtensions.swift

```diff
@@ -14,10 +14,29 @@ import AppKit
 // MARK: - Properties
 
 public extension NSAttributedString {
-    #if os(iOS)
-    /// SwifterSwift: Bolded string.
+    #if canImport(UIKit)
+    private typealias Font = UIFont
+    #endif
+
+    #if canImport(AppKit) && !targetEnvironment(macCatalyst)
+    private typealias Font = NSFont
+    #endif
+
+    /// SwifterSwift: Bolded string using the system font.
+    #if !os(Linux)
     var bolded: NSAttributedString {
-        return applying(attributes: [.font: UIFont.boldSystemFont(ofSize: UIFont.systemFontSize)])
+        let pointSize: CGFloat
+        if !string.isEmpty,
+            let font = attribute(.font, at: 0, effectiveRange: nil) as? Font {
+            pointSize = font.pointSize
+        } else {
+            #if os(tvOS) || os(watchOS)
+            pointSize = Font.preferredFont(forTextStyle: .headline).pointSize
```

Comment:
very minor, but should this be .body? same in `italicized`. It may not actually matter:

https://gist.github.com/zacwest/916d31da5d03405809c4#file-results-markdown

Reply:
This is the default in `tvOS`. In `watchOS` you can't really investigate, but it looks right.